### ISSUE TITLE
Nest Developer Data Missing

### ISSUE THREAD

**Issue by beetree** [tonesto7/nest-manager#543]
## Verify the following before opening an issue:

- [x ] You followed the Nest Developer documentation here? [Nest Developer Account Creation](https://cdn.rawgit.com/tonesto7/nest-manager/master/Documents/html/Create_Dev_Account.html)
- [x ] That _OAuth_ is Enabled for the SmartApp under the IDE.
- [x (Jan 20 update from repo) ] The SmartApps and *_ALL_* (5) Devices Handlers are using the latest code available.
- [x ] You have reviewed the [NST Manager Wiki](http://thingsthataresmart.wiki/index.php?title=NST_Manager) for potential fixes.
- [x ] That ALL SmartApps and Device Handlers have been _Published for You_ in the IDE.

## About Your Setup:
* How many Thermostats?: 2
* How many Protects? 0
* How many Cameras? 0
* Using the Nest Presence Device? (don't know what this is, I believe not)
* Using the Nest Weather Device? No
* iOS or Android?: Android
* Mobile App Version: SmartThings 2.13.0

## Expected Behavior

Error message when trying to install the "Nest Manager" SmartApp in the Mobile App: "Nest Developer Data Missing", https://github.com/tonesto7/nest-manager/blob/master/smartapps/tonesto7/nest-manager.src/nest-manager.groovy

## Current Behavior

I expected to be asked for my Nest Login details as outlined here: http://thingsthataresmart.wiki/index.php?title=NST_Manager#Installation_Instructions

## Steps to Reproduce (for bugs)

I would assume that this is unique to my setup in some way.
The only step that I could not follow exact in the instructions was which "shard" it is installed on. As per the instructions in the wiki, I go to https://consigliere-regional.api.smartthings.com/ which redirects me to graph-na04-useast2.api.smartthings.com which is also where I have installed everything. However, further down in the wiki, there's a linkto graph-na02-useast1.api.smartthings.com . I have NOT installed it on graph-na02-useast1. Please advice on how I should interpret the wiki here. Is the redirect correct, or the direct link?

## Context

Install the app

Please include a copy of your log under the IDE if possible
Happy to pull logs. Unclear how I do this (couldn't find instructions in the wiki). Under "Live Logging" in the IDE there is only this:
```
 8:32:59 AM: info Waiting on events...
 8:32:59 AM: info For past logs for individual things go to the My Devices section, find the device and click on the Events link on the device information page.
 8:32:59 AM: info This console provides live logging of your SmartThings.
```

Thank you so so much for developing this! We all depend on you to get Nest working under SmartThings. Amazing work!

**Comment by beetree** [comment]
Clearly, I need to do better reading.

If you're getting this issue, you need to connect everything with a Nest account. The instructions are here: https://cdn.rawgit.com/tonesto7/nest-manager/master/Documents/html/Create_Dev_Account.html

This wasn't really clear in the wiki (or I missed it). Consider making it more clear.

Thanks!

**Comment by brianm1962** [comment]
@beetree - Thanks for posting the instructions to add Nest Developer data.  I am try to add this information to the Nest Manager smartapp to enable it in ST. 
Right now I get the error: Nest Developer Data Missing.
I have completed steps 1 and 2 from your instructions. But when I get to step 3, and try to input the settings, I am presented with: "Add setting definitions in the source code and then set the values here".
Not sure where to go from here???

UPDATE: I stumbled across something that said NOT to AUTH Automations. I had done so. So, deleted it and reinstalled it without AUTH. This removed the block and I was able to continue to step 3. 

Now its working!!  Thanks community!

I'm totally new to all this. I was able to add my MY-Q Chamberlain garage door opener, and now NEST. Next is RING Doorbell Pro. Soon I will add some GE smarts switches and KEVO and start developing some automation. Then SONOS on my list.
